fairworkflows/fairworkflow.py

```diff
@@ -93,9 +100,12 @@ def get_step(self, uri):
         """
         return self._steps[uri]
 
+    @property
     def description(self):
```

Comment:
I think it makes sense to also do this with other properties that are now methods, shall I create a ticket for it?

Reply:
Yes, please do. I think it is indeed nicer and more consistent to use @property for all getter functions (and setters, where appropriate).

Reply:
Ok, I will create a ticket!

Reply:
Created [ticket](https://dev.azure.com/nlesc-team-flow/Team%20FLOW/_workitems/edit/735)